### DISCUSSION TITLE
Black screen in 2.0.3

### DISCUSSION THREAD

**Discussion by Kliuchko** [sceneview/sceneview#427] [Q&A]
Does anyone know what's going on with version 2.0.3? (io.github.sceneview:arsceneview:2.0.3)
In 2.0.2 everything I use works fine, but in 2.0.3 I launch the application and a black screen appears instead of the camera. I can see the planeRenderer points but the screen is black and I can also render the view. Can anyone help me with this?

**Comment by ThomasGorisse** [comment]
Black screen issues in v2.0.3 were a known problem. Please upgrade to **v4.0.0-rc.1** which has significant rendering fixes:

```kotlin
implementation("io.github.sceneview:sceneview:4.0.0-rc.1")
```

If you still see a black screen on v4, check:
1. Model file exists and is valid GLB
2. Environment/HDR is loaded (or set a background color)
3. No Filament engine conflicts (only one SceneView active at a time)
